gemrb/core/GameScript/GameScript.cpp

```diff
@@ -1845,6 +1845,8 @@ GameScript::GameScript(const ieResRef ResRef, Scriptable* MySelf,
 	strnlwrcpy( Name, ResRef, 8 );
 
 	script = CacheScript( Name, AIScript);
+	dead = false;
+	running = false;
```

Comment:
C++11 supports initing in the declaration, which is both cleaner and a Sonar warning.